### ISSUE TITLE
unpleasant noise point in low resolution groundtruth depth map

### ISSUE THREAD

**Issue by Tord-Zhang** [YoYo000/BlendedMVS#17]
I think that maybe you resize the depth map from high resolution to low resolution, So when I manage to convert the depth maps to disparity maps, there will be some unpleasant noise point in the disparity maps (point with extremely small depth).

**Comment by YoYo000** [comment]
@Tord-Zhang Yes that might be a problem. For training, such noises might not affect the training, but in testing, I think it would be better to filter them out using some high-pass filter.

**Comment by malteprinzler** [comment]
it seems like the noise in the depth maps comes from using bilinear interpolation during downsampling. would it be possible to upload the depth files with nearest neighbor downsampling applied? This would improve the depth map quality significantly and shouldn't be much effort i guess.
